### ISSUE TITLE
"Module undefined not declared as a dependency" after updates

### ISSUE THREAD

**Issue by perrin4869** [jspm/jspm-cli#1703]
I recently conducted a few updates (including `react@15.0.0`), and suddenly the static build fails to load with the following message: `Uncaught TypeError: Module undefined not declared as a dependency.` When loading without any build or bundle, it works fine. How do you troubleshoot this case? I traced down the error to these lines:

```
$__System.registerDynamic("5b", [], false, function($__require, $__exports, $__module) {
  var _retrieveGlobal = $__System.get("@@global-helpers").prepareGlobal($__module.id, null, {"process": $__require("undefined")});
```

I'm using `jspm@0.17beta12`.
Thanks!

Edit: tracked down the module that fails to load here: https://github.com/IonDen/ion.sound/blob/master/js/ion.sound.js

I sent a pull request to the registry a while back, with the following override:
```
{
  "format": "global"
}
```

**Comment by perrin4869** [comment]
I reproduced the error here: https://github.com/perrin4869/jspm_ionsound
Running instructions are in `README.md`

**Comment by guybedford** [comment]
Fixed in https://github.com/systemjs/builder/commit/ea936ba4ef3ab8709882a95b58cff94952c4de31.

**Comment by perrin4869** [comment]
Cool, thanks! This will be included in the next beta?

**Comment by guybedford** [comment]
Released in 0.17.0-beta.13.
